packages/thirdweb/src/utils/domains.ts

```diff
@@ -90,3 +90,13 @@ export const getThirdwebBaseUrl = (service: keyof DomainOverrides) => {
   }
   return `https://${origin}`;
 };
+
+let serviceKey: string | null = null;
+
+export const setServiceKey = (key: string | null) => {
+  serviceKey = key;
+};
+
+export const getServiceKey = () => {
+  return serviceKey;
+};
```

Comment:
Should this just be a param of createThirdwebClient, since we need that when using the SDK in any service anyway?